apache/couchdb-fauxton
# Remove dependencies to pouchdb packages

## Overview

Replace `pouchdb-*` packages with our own implementation to simplify the dependency tree. 
The motivation is that some of these packages haven't had new releases in years, which is a problem when a sub-dependency requires security fixes.

## Testing recommendations

- Open a document with conflicts in the editor
- Click the "Conflicts" button
- Make sure you can compare the different revisions

## GitHub issue number

n/a

## Related Pull Requests

n/a

## Checklist

- [ ] Update [rebar.config.script](https://github.com/apache/couchdb/blob/main/rebar.config.script) with the correct tag once a new Fauxton release is made
